src/thread.c

```diff
@@ -2217,6 +2217,28 @@ void ABTI_thread_print(ABTI_thread *p_thread, FILE *p_os, int indent)
     ABTU_free(prefix);
 }
 
+int ABTI_thread_stack_print(ABTI_thread *p_thread, FILE *p_os) {
+    void *p_stack = p_thread->attr.p_stack;
+    size_t i, stacksize = p_thread->attr.stacksize;
```

Comment:
I think it is also important to print the ULT's current ABTD_thread_context/abt_ucontext_t here, and particularly (struct abt_ucontext_t).fctx, if defined(ABT_CONFIG_USE_FCONTEXT), or (ucontext_t).uc_mcontext.gregs[REG_RSP] otherwise.
This will allow to detect hung threads and also help for manual unwinding for now.

Reply:
Thank you for suggestion. We will show the context data for now if it is helpful.

Reply:
Thank you, but do you agree with me that it could be helpful to get the current context in order to know where we are in each ULT's stack, and also it can indicate progress or not.

Reply:
1. fcontext
Yes. `abt_ucontext_t->fctx` should point to the bottom of the call stack (and that function should be ` jump_fcontext` at present) if the ULT has suspended. It should be helpful to unwind the call stack.
2. ucontext
I am not sure, but `[REG_RSP]` looks helpful. I note that the current Argobots lacks the ucontext support (#20).